test/e2e/crd.go

```diff
@@ -103,14 +102,13 @@ func validateCrdCrud(f framework.FederationFramework, targetCrdKind string, name
 	userAgent := fmt.Sprintf("test-%s-crud", strings.ToLower(targetCrdKind))
 
 	// Create the target crd in all clusters
-	var pools []dynamic.ClientPool
-	var hostPool dynamic.ClientPool
+	var configs []*rest.Config
+	var config *rest.Config
```

Comment:
s/config/hostConfig/ for clarity.

Reply:
Oh, that's because they are the same value.  Please remove the `config` variable and ensure instances of `hostPool` are changed to `hostConfig`.

Reply:
have done :-)